.eslintrc.json

```diff
@@ -102,20 +102,20 @@
         ]
       }
     },
-    {
-      "files": ["*.spec.ts"],
-      "extends": ["plugin:jest/recommended"],
-      "rules": {
-        "jest/expect-expect": "off"
-      }
-    },
     {
       "files": ["*.html"],
       "rules": {}
     },
     {
       "files": ["*.ts", "*.js"],
       "extends": ["prettier"]
+    },
+    {
+      "files": ["*.spec.ts"],
+      "extends": ["plugin:testing-library/angular"],
```

Comment:
`"plugin:testing-library/angular"` can be used by _all_ projects. Therefore added to the main eslint.json